crates/web-plugins/oob-messages/src/web.rs

```diff
@@ -4,18 +4,24 @@ use crate::{
     web::handler::{handler_landing_page_oob, handler_oob_inv, handler_oob_qr},
 };
 use axum::{routing::get, Router};
+use handler::decode_oob_inv;
 use std::sync::Arc;
 
 pub(crate) fn routes(state: Arc<OOBMessagesState>) -> Router {
+    let invitation_path = format!("/_{}", &state.oobmessage);
     Router::new() //
         .route("/oob_url", get(handler_oob_inv))
         .route("/oob_qr", get(handler_oob_qr))
         .route("/", get(handler_landing_page_oob))
+        // handle oob invitation to invitation message
+        .route(&invitation_path, get(decode_oob_inv))
```

Comment:
I dont understand what the new route is and why you added it

Reply:
The qr_code and and oob url have no handled route, what i mean is if you can the oob qr_code you get nothing even the oob_url takes you no where as the route has not been handled and as such you can have its content without decoding it, that's where this route comes in